Clarify binding options and remove mention of deny_null_bind (#1337)
hashicorp/web-unified-docs: content/vault/v1.16.x/content/docs/auth/ldap.mdx

### Binding parameters

The LDAP auth method supports the following methods for resolving the user object used to authenticate the end user:

> - **Search** - Searches the LDAP server directory for the user object based on the provided username. This search can performed in one of two ways:

Reviewer: ```suggestion
- **Search** - Searches the LDAP server directory for the user object based on the provided username. You can search in two ways:
```
Style correction: avoid "this" as a pronoun
